lab/src/mapreduce/common_map.go

```diff
@@ -53,6 +58,42 @@ func doMap(
 	//
 	// Your code here (Part I).
 	//
+	fileByte, err := ioutil.ReadFile(inFile)
+	checkError(err)
+
+	textContent := string(fileByte)
+	mapResults := mapF(inFile, textContent)
+
+	splitResults := make(map[int][]KeyValue)
+
+	for _, keyValuePair := range mapResults {
+		reduceIndex := int(math.Mod(float64(ihash(keyValuePair.Key)), float64(nReduce)))
```

Comment:
[nit] Is it possible to not translate the hash value and `nReduce` between `float64` and `int` back and forth?